### PR TITLE
remove changes of MAGETWO-69567

### PR DESCRIPTION
**Pull request by torhoehn**
Undo changes of MAGETWO-69567

### Description
Previous changes in the fotorama gallery plugin causes problems on iOS devices.

### Fixed Issues (if relevant)
1. magento/magento2#13226: Can't close full screen gallery on iOS devices

### Manual testing scenarios
1. Start Apple Simulator (https://developer.apple.com/library/content/documentation/IDEs/Conceptual/iOS_Simulator_Guide/Introduction/Introduction.html)
2. Run Safari and open any product detail page
3. Tap on the image to open the product image in fullscreen mode
4. Tap on the close button in the upper right corner
5. Gallery closes and you can see the product detail page again

### Contribution checklist
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Review by iivashchenko — COMMENTED** [review]
Removing this changes will provoke and additional click triggered and, as a result, multi-cart or search-box will be exposed. There is a resolved  MAGETWO-85708 ticket where a special case for IOS devices was added.

**Comment by torhoehn** [comment]
Maybe it would be a good idea to make your internal tickets or the associated pull request public.

**Comment by iivashchenko** [comment]
@torhoehn Sorry, I can't make an influence on that, but I can inform you when mentioned ticket will be added to the public pull request or finally merged.

**Comment by miguelbalparda** [comment]
There are some [commits](https://github.com/magento/magento2/search?q=+MAGETWO-85708&type=Commits&utf8=%E2%9C%93) with that internal ticket number. Can you review them and confirm we can close this PR?

**Comment by iivashchenko** [comment]
@miguelbalparda yes, this PR can be closed.

**Comment by anastaciyagavrilova** [comment]
Hi,

The fix is still not working on Magento ver. 2.2.3 in /html/lib/web/fotorama/fotorama.js file

  if (o_allowFullScreen) {
                $fullscreenIcon.prependTo($stage);
                o_nativeFullScreen = FULLSCREEN && o_allowFullScreen === 'native';

                // Due 300ms click delay on mobile devices
                // we stub touchend and fallback to click.
                // MAGETWO-69567

                stubEvent($fullscreenIcon, 'touchend');
            } else {
                $fullscreenIcon.detach();
                o_nativeFullScreen = false;
            }
IN iphones Cross Button is not working and gallery is not closing.
Any solution:--

**Comment by iivashchenko** [comment]
@anastaciyagavrilova Thank you for the report. It seems the fix is not ported to 2.2.3. As a solution, you can create and apply patch from [StubEvent function of Magento 2.2.2](https://github.com/magento/magento2/blob/2.2-develop/lib/web/fotorama/fotorama.js#L1220-L1234). Internal ticket MAGETWO-88973 to port the fix was created.

**Comment by h2ojunkie** [comment]
The link posted for  StubEvent function of Magento 2.2.2. is 404'ing.   Is there another place to get this fix?

**Comment by iivashchenko** [comment]
@h2ojunkie sorry for the broken link. Right address is [fotorama.js](https://github.com/magento/magento2/blob/2.2-develop/lib/web/fotorama/fotorama.js#L1220-L1234)

**Comment by PatelHemali** [comment]
@h2ojunkie.......................Thanks a Lot  I got Solution for the same issue..........

**Comment by PatelHemali** [comment]
@iivashchenko ...............Thanks a lot.........

**Comment by Brengineer** [comment]
This issue applies to all touch devices, not just iOS. For example, Chrome on Android has the same issue. (The Fotorama gallery does not close when tapping the button.)

**Comment by Brengineer** [comment]
@iivashchenko - I have not been able to reproduce the issue that MAGETWO-69567 claims to fix, and I can't find the original issue, so I can't see what the reproducible steps are.
What I do know is that the changes it included are causing the issue described here, and it is not limited to iOS, so [commit d50c512](https://github.com/magento/magento2/commit/d50c5126df3202a7952cb65f30aed21af9ac67d5) (for MAGETWO-85708) is not sufficient.
I have a patch that I have used on multiple projects that directly reverts [commit dd9d57f](https://github.com/magento/magento2/commit/dd9d57f170282900b32513ed4c7142f81cd413e1), and I have not encountered any problems, so as far as I can tell, the best solution is to do exactly as the title of this PR says and remove changes of MAGETWO-69567.
If you think it's still an issue, please inform us of the reproducible steps for MAGETWO-69567 on the current version of Magento, so we can find a solution that will successfully close the image gallery on all touch devices without ever activating the minicart or search box.

**Comment by iivashchenko** [comment]
@Brengineer Hi, and thank you for your feedback. We didn't get gallery issues connected with mobile devices, excluding IOS, yet. If you have such issue - please create the corresponding ticket and provide accurate information about your device and Magento version. Also, we have no reported issues with gallery closing since [dd9d57f](https://github.com/magento/magento2/commit/dd9d57f170282900b32513ed4c7142f81cd413e1). Just FYI, the original steps to reproduce MAGETWO-69567 are:
1. Open developer tools > emulate a mobile phone device
2. Go to site front end > any product
3. Click on the main image to enlarge
4. Notice image window opens
5. Click the X button to close the window
6. Notice depending on where the button is clicked either the search bar or the mini cart drop down opens.
